studyhub/base/views.py

```
@@ -1,14 +1,22 @@
+from django.db.models import Q
+from .models import Room, Topic
 from .forms import RoomForm
 
 # Create your views here.
 
 
 
 def home(request):
-    rooms = Room.objects.all()
-    context = {'rooms': rooms}
+    q = request.GET.get('q') if request.GET.get('q') != None else ''
```

Ruff

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)